<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="jenkins-agent-10-25-220-208" name="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" tests="6" failures="0" timestamp="2022-10-27T10:41:14 UTC" time="60.346" errors="0">
  <testcase name="invokeAsyncTestWith_429" time="5.055" classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest"/>
  <system-out/>
  <testcase name="invokeAsyncTestWith_503" time="40.091" classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest"/>
  <system-out/>
  <testcase name="invokeAsyncTestWith_502" time="13.054" classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest"/>
  <system-out/>
  <testcase name="invokeAsyncTestWith_504_And_No_MaxRetryTime" time="0.053" classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest"/>
  <system-out/>
  <testcase name="invokeAsyncTestWith_429_And_No_MaxRetryTime" time="0.036" classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest"/>
  <system-out/>
  <testcase name="invokeAsyncTestWith_504" time="2.057" classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest"/>
  <system-out/>
</testsuite> <!-- com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest -->
